<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Autotest::Internal::GTestSettingsPage</class>
 <widget class="QWidget" name="Autotest::Internal::GTestSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QCheckBox" name="breakOnFailureCB">
         <property name="toolTip">
          <string>Turns failures into debugger breakpoints.</string>
         </property>
         <property name="text">
          <string>Break on failure while debugging</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="runDisabledGTestsCB">
         <property name="toolTip">
          <string>Executes disabled tests when performing a test run.</string>
         </property>
         <property name="text">
          <string>Run disabled tests</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="throwOnFailureCB">
         <property name="toolTip">
          <string>Turns assertion failures into C++ exceptions.</string>
         </property>
         <property name="text">
          <string>Throw on failure</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Iterations:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="shuffleGTestsCB">
         <property name="toolTip">
          <string>Shuffles tests automatically on every iteration by the given seed.</string>
         </property>
         <property name="text">
          <string>Shuffle tests</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="repeatGTestsCB">
         <property name="toolTip">
          <string>Repeats a test run (you might be required to increase the timeout to avoid canceling the tests).</string>
         </property>
         <property name="text">
          <string>Repeat tests</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="repetitionSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>9999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Seed:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QSpinBox" name="seedSpin">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>A seed of 0 generates a seed based on the current timestamp.</string>
         </property>
         <property name="specialValueText">
          <string/>
         </property>
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Group mode:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Active filter:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QComboBox" name="groupModeCombo">
         <property name="toolTip">
          <string>Select on what grouping the tests should be based.</string>
         </property>
         <item>
          <property name="text">
           <string>Directory</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GTest Filter</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="Utils::FancyLineEdit" name="filterLineEdit">
         <property name="toolTip">
          <string>Set the GTest filter to be used for grouping.
See Google Test documentation for further information on GTest filters.</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>60</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::FancyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">utils/fancylineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
